src/components/Canvas.tsx

```diff
@@ -17,6 +17,19 @@ import {
 } from "volglass-backend";
 import { match, P } from "ts-pattern";
 
+// The class name has to be defined without interpolation at compile time
+// so tailwind will be able to see and not strip it from production css
+// TODO: support edge colors
+const NODE_STYLES = {
+	"0": "bg-gray-500/50 border-gray-500",
+	"1": "bg-red-600/50 border-red-600",
+	"2": "bg-orange-500/50 border-orange-500",
+	"3": "bg-yellow-300/50 border-yellow-300",
+	"4": "bg-green-400/50 border-green-400",
+	"5": "bg-cyan-400/50 border-cyan-400",
+	"6": "bg-violet-500/50 border-violet-500",
+}
+
```

Comment:
This could be put to use.
https://tailwindcss.com/docs/customizing-colors#arbitrary-values

However, I don't know how obsidian calculate background color from hex. Do you know information about this?